_source/user-guide/infrastructure-monitoring/p8s-remote-write.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: article
+title: Remote write for Prometheus
+permalink: /user-guide/metrics/p8s-remote-write.html
+flags:
+  logzio-plan: community
+tags:
+  - metrics integrations
+contributors:
+  - yberlinger
+---
+This topic is for Prometheus users migrating their solution to Logz.io Infrastructure Monitoring powered by Grafana.
+
+You can send your Prometheus metrics to your Logz.io Infrastructure Monitoring account.
+To do this, you’ll need to send your application metrics to Logz.io in JSON format and stick to the formatting guidelines.
```

Comment:
I don't think we should put this here, maybe confusing (the JSON format note)
the user just defines the remote write, we are in charge od the format that gets to the listener